src/Serializers/FingerprintSerializer.php

```diff
@@ -57,40 +71,16 @@ private function addIdToSerialization( EntityId $id = null, array &$serializatio
 		$serialization['id'] = $id->getSerialization();
 	}
 
-	private function addLabelsToSerialization( TermList $labels, array &$serialization ) {
-		$serialization['labels'] = $this->serializeValuePerTermList( $labels );
-	}
-
+	/**
+	 * @deprecated this is used somewhere stupid...
```

Comment:
I believe bene is preparing a commit on top of this that will simply kill the fingerprint stuff